templates/download/_nav_secondary-v1.html

```diff
@@ -1,9 +1,8 @@
-  <ul{% if list_class %} class="{{ list_class }}"{% endif %} class="hover-menu">
-    <li><a{% if level_1 == 'download' and not level_2 %} class="is-active"{% endif %} href="/download">Overview</a></li>
-    <li><a{% if level_2 == 'cloud' %} class="is-active"{% endif %} href="/download/cloud">Cloud</a></li>
-    <li><a{% if level_2 == 'server' %} class="is-active"{% endif %} href="/download/server">Server</a></li>
-    <li><a{% if level_2 == 'desktop' %} class="is-active"{% endif %} href="/download/desktop">Desktop</a></li>
-    <li><a{% if level_2 == 'ubuntu-kylin' or level_2 == 'ubuntu-kylin-zh-CN' %} class="is-active"{% endif %} href="/download/ubuntu-kylin">Ubuntu Kylin</a></li>
-    <li><a{% if level_2 == 'alternative-downloads' %} class="is-active"{% endif %} href="/download/alternative-downloads">Alternative downloads</a></li>
-    <li><a{% if level_2 == 'ubuntu-flavours' %} class="is-active"{% endif %} href="/download/ubuntu-flavours">Ubuntu flavours</a></li>
-  </ul>
+<ul class="{% if list_class %}{{ list_class }}{% else %}nav-secondary__menu p-inline-list{% endif %}">
+  <li{% if not list_class %} class="p-inline-list__item"{% endif %}><a class="{% if not list_class %}p-inline-list__link {% endif %}{% if level_1 == 'download' and not level_2 %} is-active"{% endif %}" href="/download">Overview</a></li>
```

Comment:
Is `{% if not list_class %}` required? I didn't add these on the other section secondary navs?